lib/screens/chat_rooms_screen.dart

```diff
@@ -1,13 +1,50 @@
 import 'package:flutter/material.dart';
+import '../widgets/search_widget.dart';
+import '../helper/authenticate_helper.dart';
+import '../services/auth.dart';
+import '../configuration/app_colors.dart';
+import '../configuration/app_path.dart';
 
 class ChatRoom extends StatefulWidget {
   @override
   _ChatRoomState createState() => _ChatRoomState();
 }
 
 class _ChatRoomState extends State<ChatRoom> {
+  AuthMethods authMethods = AuthMethods();
+
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return Scaffold(
+      appBar: AppBar(
+        title: Image.asset(
+          AssetsPath.logoAsset,
+          height: 40,
+          color: AppColors.primaryIconColor,
+        ),
+        actions: [
+          GestureDetector(
+            onTap: () {
+              authMethods.SignOut();
+              Navigator.pushReplacement(
+                  context,
+                  MaterialPageRoute(
+                    builder: (context) => Authenticate(),
+                  ));
+            },
+            child: Container(
+                padding: EdgeInsets.symmetric(horizontal: 16),
+                child: Icon(Icons.exit_to_app)),
+          )
+        ],
+      ),
+      floatingActionButton: FloatingActionButton(
+        child: Icon(Icons.search),
+        onPressed: () {
```

Comment:
you can use arrow function here. If you have only one function inside another function it could be simlified to () => Navigator.push();